CONTRIBUTING.md

```diff
@@ -1,23 +1,83 @@
 Contributing to the openPMD standard
 ======================================
 
-Participation
--------------
+The openPMD standard can evolve in order to accommodate the needs of the community. This results in successive [*versions* of the standard](https://github.com/openPMD/openPMD-standard/blob/1.0.1/STANDARD.md#the-versions-of-this-standard). This document explains the process by which the standard evolves, and how to contribute to it.
+
+Update Cycle
+------------
 
-We participate via *pull requests* on GitHub.
-Improvements will be discussed and adressed in those.
+The updates of the openPMD standard go through a 3-step process:
 
-Future proposals and strategies can be discussed in GitHub *issues*.
+- **Issues:** Anyone can propose changes to the standard, request new features,
+ask for clarification, etc. by opening a new issue
+[here](https://github.com/openPMD/openPMD-standard/issues). The proposed
+changes/features/question will then be discussed by the members of the
+openPMD community (or anyone interested, really) in the `Comments` section of
+this issue. The openPMD admins will usually add a label specifying the version
+of the standard in which this should be implemented.
 
+- **Pull requests:** Once an issue is well-understood and a clear solution
+emerges, anyone can volunteer to implement it in the text of the standard. This
+is done by creating a pull request (see below for more details on how to
+create a pull request). Pull requests are then reviewed by the community, and
+eventually merged by the openPMD admins (@ax3l and @RemiLehe) into the
+`upcoming-*` branch (where `*` is replaced by the number of the next upcoming
+version), so that the corresponding changes will be included in the next release.
 
-Update Cycle
-------------
+- **New releases:** Once all the issues that have been labeled for the next
+upcoming version have been resolved (e.g. via pull requests), the openPMD
+admins will create a new official version (by merging the `upcoming-*` on top
+of the `latest` branch and tagging it with a new **official version number**.)
+Tools that use the openPMD standard should then be adapted, so that they can
+properly parse openPMD files that conform to this new official version.
+
+Note that this 3-step process is reflected in the [Projects
+tab](https://github.com/openPMD/openPMD-standard/projects). This page
+lists the upcoming versions that are being considered, and, for each
+upcoming version, tracks the corresponding issues and their status (from
+`proposed` to `accepted` to `implemented`). The Projects tab is important for
+the prioritization and organization of tasks.
```

Comment:
Mention that issues can be postponed for later releases, which is *not* a bad thing.
It just means the solution is not yet fully emerged, there are alternative approaches that might need further investigation or simply lacks time to implement in some cases.